src/app/components/service/service.component.ts

```diff
@@ -87,14 +89,16 @@ export class ServiceComponent implements OnInit, OnDestroy {
 
   translations$ = signal<Translation[]>([])
 
-  filteredTranslations$: Signal<Translation[]> = signal([])
+  filteredTranslations$!: Signal<Map<string, Message[]>>
```

Comment:
Private? By default, you can make properties private. And make them public when needed.

```suggestion
  private filteredTranslations$!: Signal<Map<string, Message[]>>
```

Reply:
Templates need public properties in the component. Anything else can be private.